### PR TITLE
Implements PASSENGER AT STOP TIME and TimetabledTime constraints

### PR DESCRIPTION
**Pull request by skinkie**
@nick-knowles this is the kind of pull request I want to see.

As you can see it is directly based on next. The other pull request (recharging) has not been merged. The feature is completely independent.

### CODE REVIEW

File: xsd/netex_part_2/part2_journeyTimes/netex_passengerAtStopTime_version.xsd [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+				<xsd:sequence>
+					<xsd:group ref="PassengerAtStopTimeGroup"/>
+				</xsd:sequence>
+			</xsd:extension>
+		</xsd:complexContent>
+	</xsd:complexType>
+	<xsd:element name="PassengerAtStopTime" abstract="false" substitutionGroup="PassingTime">
+		<xsd:annotation>
+			<xsd:documentation>Time at which a passenger should be at a stop to commence or finish a process. +v2.0
+</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_2/part2_journeyTimes/netex_passengerAtStopTime_version.xsd:91]:
```suggestion
			<xsd:documentation>Time at which a passenger should be at a stop to commence or finish a process. +v2.0 </xsd:documentation>
```

File: xsd/netex_part_2/part2_journeyTimes/netex_passengerAtStopTime_version.xsd [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+Air transport, Airports,
+Ports and maritime transport, Ferries (marine),
+Public transport, Bus services, Coach services, Bus stops and stations,
+Rail transport, Railway stations and track, Train services, Underground trains,
+Business and industry, Transport, Air transport , Ports and maritime transport, Public transport,
+Rail transport, Roads and Road transport
+</Category>
+					<Project>CEN TC278 WG3 SG9.</Project>
+				</Subject>
+				<Title>NeTEx PASSENGER AT STOP TIME    types.</Title>
```
Comment (skinkie) [inline xsd/netex_part_2/part2_journeyTimes/netex_passengerAtStopTime_version.xsd:57]:
```suggestion
				<Title>NeTEx PASSENGER AT STOP TIME types.</Title>
```

File: xsd/netex_part_2/part2_journeyTimes/netex_passengerAtStopTime_version.xsd [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+		</xsd:complexType>
+	</xsd:element>
+	<xsd:group name="PassengerAtStopTimeGroup">
+		<xsd:annotation>
+			<xsd:documentation>PASSENGER AT STOP TIME elements.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:sequence>
+			<xsd:element name="Description" type="MultilingualString" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Description of tPASSEMGER at STOP time.</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_2/part2_journeyTimes/netex_passengerAtStopTime_version.xsd:121]:
```suggestion
					<xsd:documentation>Description of PASSENGER AT STOP TIME.</xsd:documentation>
```

File: xsd/netex_part_2/part2_journeyTimes/netex_passengerAtStopTime_version.xsd [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="EarliestTime" type="xsd:time" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Earliest  time that passenger should be at stop for specified process, e.g. baggage check in, customs etc.
+</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_2/part2_journeyTimes/netex_passengerAtStopTime_version.xsd:127]:
```suggestion
					<xsd:documentation>Earliest time that passenger should be at stop for specified process, e.g. baggage check in, customs etc.</xsd:documentation>
```

File: xsd/netex_part_2/part2_journeyTimes/netex_passengerAtStopTime_version.xsd [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="EarliestDayOffset" type="DayOffsetType" default="0" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Number of days after the starting time of the journey if  not same calendar day. Default is 0 for same day.</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_2/part2_journeyTimes/netex_passengerAtStopTime_version.xsd:132]:
```suggestion
					<xsd:documentation>Number of days after the starting time of the journey if not same calendar day. Default is 0 for same day.</xsd:documentation>
```

File: xsd/netex_part_2/part2_journeyTimes/netex_passengerAtStopTime_version.xsd [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="LatestTime" type="xsd:time" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Latest  time that passenger should be at stop for specifoed process.</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_2/part2_journeyTimes/netex_passengerAtStopTime_version.xsd:137]:
```suggestion
					<xsd:documentation>Latest time that passenger should be at stop for specified process.</xsd:documentation>
```

File: xsd/netex_part_2/part2_journeyTimes/netex_passengerAtStopTimes_support.xsd [hunk 7]
```diff
@@ -0,0 +1,219 @@
...
+Air transport, Airports,
+Ports and maritime transport, Ferries (marine),
+Public transport, Bus services, Coach services, Bus stops and stations,
+Rail transport, Railway stations and track, Train services, Underground trains,
+Business and industry, Transport, Air transport , Ports and maritime transport, Public transport,
+Rail transport, Roads and Road transport
+</Category>
+					<Project>CEN TC278 WG3 SG9.</Project>
+				</Subject>
+				<Title>NeTEx PASSING TIME identifier  types.</Title>
```
Comment (skinkie) [inline xsd/netex_part_2/part2_journeyTimes/netex_passengerAtStopTimes_support.xsd:57]:
```suggestion
				<Title>NeTEx PASSING TIME identifier types.</Title>
```

**Comment by skinkie** [comment]
An example will appear once Deckplan is redone.

**Review by ue71603 — APPROVED** [review]

File: xsd/netex_part_2/part2_journeyTimes/netex_vehicleJourney_version.xsd [hunk 8]
```diff
@@ -544,7 +552,7 @@ Rail transport, Roads and Road transport
 	</xsd:complexType>
 	<xsd:group name="VehicleTypeStopAssignmentGroup">
 		<xsd:annotation>
-			<xsd:documentation>Elements for VEHICLE TYPE STOP  ASSIGNNMENT Group.</xsd:documentation>
+			<xsd:documentation>Elements for VEHICLE TYPE STOP  ASSIGNMENT Group.</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_2/part2_journeyTimes/netex_vehicleJourney_version.xsd:555]:
```suggestion
			<xsd:documentation>Elements for VEHICLE TYPE STOP ASSIGNMENT Group.</xsd:documentation>
```

**Comment by ue71603** [comment]
@Aurige pls review

**Comment by ue71603** [comment]
@skinkie  ready to merge.

**Pull request merged**